python_modules/dagster/dagster/core/execution/api.py

```diff
@@ -356,6 +359,61 @@ def ephemeral_instance_if_missing(
             yield ephemeral_instance
 
 
+def execute_job(
+    job: ReconstructableJob,
+    instance: "DagsterInstance",
+    run_config: Any = None,
+    tags: Optional[Dict[str, Any]] = None,
+    raise_on_error: bool = True,
+    op_selection: Optional[List[str]] = None,
+) -> ExecuteJobResult:
+    """Execute a job synchronously.
+
+    Users will typically call this API when testing job execution, or running standalone
+    scripts.
+
+    Parameters:
+        job (ReconstructableJob): The job to execute.
+        instance (DagsterInstance): The instance to execute against.
+        run_config (Optional[dict]): The configuration that parametrizes this run, as a dict.
+        tags (Optional[Dict[str, Any]]): Arbitrary key-value pairs that will be added to run logs.
+        raise_on_error (Optional[bool]): Whether or not to raise exceptions when they occur.
+            Defaults to ``True``, since this is the most useful behavior in test.
```

Comment:
I'm not sure we want this default here? `execute_in_process` is test focused now so this is more likely script / "real" usage where we may want to default the other way

Reply:
that makes sense to me